### DISCUSSION TITLE
Plugin Grouping

### DISCUSSION THREAD

**Discussion by MoD21k** [runelite/runelite#15701] [Ideas]
Is it possible to make groups of plugins
not just tag them with a star but be able to make a group and also collapse them

**Comment by owlfredo** [comment]
I would also like this! A default standard grouping and ability to custom group as well.

Example: 

- Skilling > Agility > all the agility related things, etc
- Raids> ToA/TOB/CoX > all their related plugins. 
- chat > etc

**Comment by AbigailCKilby** [comment]
> Is it possible to make groups of plugins not just tag them with a star but be able to make a group and also collapse them

I want to also suggest this, It would be nice for multiple reasons: 

Firstly it would be great for switching between activities such as PvM and Skilling so you don't have your PvM Plugins all over the screen while you're chilling out. 
Second, it would be nice for switching between playing the game, and creating videos/recording. Sometimes I have to turn on and off 20+ plugins just to switch into video creating mode without tiles marked and GUIs everywhere and such, and turning on preferred graphics enhancing or disabling plugins.  

What would be even doper was if there was an ability to add tiny little switches to the side of your inventory in game or something, like a layover plugin group switch, then you don't have to go into the entire settings panel and turn them on/off, sometimes compromising your OBS recording frame, etc. 

<3

**Reply by antero111** [reply]
See Plugin Presets plugin on the plugin hub. With that plugin you can make presets for your video making and pvm scenarios etc...

**Comment by Ben-Colwell** [comment]
Not sure if this is doable, but I shall try :D

**Reply by MoD21k** [reply]
This would be awsome

I was thinking of a tab that plugins can be added to even more awsome would be wenn the whole tab could toggle the ones on and off in this tab.

What if nothing in this direction would work can we get the otion to set tags by our self?
this way we atleast could group them a little bit

**Reply by Ben-Colwell** [reply]
@MoD21k See antero111's post above. The Plugin Presets plugin does basically what you want, without having to overwrite the base client.
